### PR TITLE
Add getTasksWithManagerId

### PR DESCRIPTION
**Pull request by anshpand**

### CODE REVIEW

**Review by saitejaboggarapu — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -26,6 +26,7 @@ dependencies {
 	runtimeOnly 'mysql:mysql-connector-java'
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	implementation group: 'org.json', name: 'json', version: '20220924'
```
Comment (satheeshkumarvadivel) [inline build.gradle]: Do we need to use this? Can we do it with java classes instead?

File: src/main/java/com/example/demo/rest/TaskResource.java [hunk 2]
```diff
@@ -57,7 +58,65 @@ public ResponseEntity<?> getUserTasks(@PathVariable("userId") int userId, @Reque
         user.get().setTasks(tasks);
         return new ResponseEntity<>(user.get().getUserWithTasks(), HttpStatus.OK);
     }
+    @GetMapping("/managerId")
```
Comment (satheeshkumarvadivel) [inline src/main/java/com/example/demo/rest/TaskResource.java]: Can we change this to "/api/v1/users/{userId}/teamtasks" and get the managerId from the provided userId?
Extract to a different Resource class if needed.

File: src/main/java/com/example/demo/rest/TaskResource.java [hunk 3]
```diff
@@ -57,7 +58,65 @@ public ResponseEntity<?> getUserTasks(@PathVariable("userId") int userId, @Reque
...
+        String managerEmailId = user.get().getManagerEmailId();
+        List<Map<String, Object>> usersDataWithManagerId = userInfoDao.getUserInfoByManagerEmailId(managerEmailId, fromDate, toDate);
+        JSONObject userData = new JSONObject();
+
+        for(Map<String, Object> userWithManagerId:usersDataWithManagerId){
 
+            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MMM-dd");
+            Date createdDate = (Date) userWithManagerId.get("created_date");
+
+            JSONObject task = new JSONObject();
```
Comment (satheeshkumarvadivel) [inline src/main/java/com/example/demo/rest/TaskResource.java]: Let's use java classes instead of JSONObject

**Review by satheeshkumarvadivel — CHANGES_REQUESTED** [review]

**Review by satheeshkumarvadivel — APPROVED** [review]
Looks good

**Pull request merged**